src/test/java/com/epam/ui/BaseTest.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.ui;
+
+import com.codeborne.selenide.testng.ScreenShooter;
+import com.epam.page.FilterPage;
+import com.epam.page.WelcomePage;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+import static com.codeborne.selenide.Selenide.closeWebDriver;
+import static com.codeborne.selenide.Selenide.open;
+import static com.codeborne.selenide.WebDriverRunner.getWebDriver;
+import static com.epam.constant.Constants.FILTERS_PAGE_URL;
+import static com.epam.constant.Constants.SIGNED_IN_NOTIFICATION;
+
+public class BaseTest {
+    private final Logger logger = LogManager.getLogger(BaseTest.class);
+    private final WelcomePage welcomePage = new WelcomePage();
+    private final FilterPage filterPage = new FilterPage();
+
+    @BeforeTest
+    public void openChromeBrowserByRPFiltersUrl() {
+        ScreenShooter.captureSuccessfulTests = false;
+        openFilterPage();
+        getWebDriver().manage().window().maximize();
```

Comment:
you can configure this via `Configuration` by setting default parameters for the driver.